code/script.js

```diff
@@ -26,22 +42,221 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  // Scroll to last message when there are too many in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// The dialogue starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hi there! What's your name?`, 'bot')
+}
+
+const moodQuestion = () => {
+  
+  showMessage(`That's a nice name! How are you feeling today?`, 'bot')
+
+  // Generate a set of buttons with mood options
+
+  inputWrapper.innerHTML = `
+  <button id="sadBtn">Sad</button>
+  <button id="neutralBtn">Neutral</button>
+  <button id="worriedBtn">Worried</button>
+  <button id="happyBtn">Happy</button>
+  `
+  
+  // Send to next question depending on which button was pushed
+
+  document.getElementById('sadBtn').addEventListener('click', () => followUpQuestion('sad'))
+  document.getElementById('neutralBtn').addEventListener('click', () => followUpQuestion('neutral'))
+  document.getElementById('worriedBtn').addEventListener('click', () => followUpQuestion('worried'))
+  document.getElementById('happyBtn').addEventListener('click', () => followUpQuestion('happy')) 
+
 }
 
-// Set up your eventlisteners here
+const followUpQuestion = (mood) => {
+  
+  showMessage(`I\m feeling ${mood}.`, 'user')
+  showMessage(`You\'re feeling ${mood}?`, 'bot')
+
+    if (mood === 'sad'){
+      
+      showMessage('I\'m sorry to hear that. Would you like to talk to someone?', 'bot')
+
+      // Generate a drop-down menu with binary options
+
+      inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>Select an option 👇</option>
+        <option value="yes">Yes</option>
+        <option value="no">No</option>
+      </select>
+    `
+    // Send to next question depending on selected option
+
+    const select = document.getElementById('select')
+    select.addEventListener('change', handleSadActivity)
+  } else if (mood === 'neutral') {
+    showMessage('That\'s good. Wanna see some cute animal pics?', 'bot')
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>Select an option 👇</option>
+        <option value="yes">Yes</option>
+        <option value="no">No</option>
+      </select>
+    `
+    const select = document.getElementById('select')
+    select.addEventListener('change', handleNeutralActivity)
+  }  else if (mood === 'worried') {
+    showMessage('That must be tough. Would you like to do a guided meditation?', 'bot')
+    inputWrapper.innerHTML = `
```

Comment:
Maybe a next step would be to remove the innerHTML when user has made its choice. Because now you can press yes, and then no, and then yes and so on.

Reply:
Indeed, good catch! We got the same feedback from Jennie.